task/bq2bq/upstream/schema.go

```diff
@@ -66,14 +66,34 @@ func ReadSchemasUnderGroup(ctx context.Context, client bqiface.Client, group *Re
 }
 
 func buildQuery(group *ResourceGroup) string {
-	modifiedNames := make([]string, len(group.Names))
-	for i, n := range group.Names {
-		modifiedNames[i] = "'" + n + "'"
+	var nameQueries, prefixQueries []string
+	for _, n := range group.Names {
+		suffix := "*"
```

Comment:
nit: Can we move this to constants ? I don't see it changing anywhere

Reply:
sure